test/loader_test.rb

```diff
@@ -99,6 +99,19 @@ class Application < Lotus::Application; load!; end
 
           output.must_match(/BeerShop/)
         end
+
+        it 'supports configured custom logger' do
+          class MyLogger < Logger; end
```

Comment:
@rpanachi i'd prefer that you declare the new logger class in `before` and killing it in `after`. The same goes for the `DrinkShop:: Application`